### DISCUSSION TITLE
Source-map does not show column mapping normally

### DISCUSSION THREAD

**Discussion by spicylemonhaha** [webpack/webpack#14988] [Q&A]
# Bug report
**What is the current behavior?**
In the development mode, if devtool is configured as source-map, the packaged file will not generate column mapping information normally if there is an error, but if the simplest babel-loader is configured, there will be normal column mapping information.

**If the current behavior is a bug, please provide the steps to reproduce.**

To reproduce the specific error, please read the sample repository I created. https://github.com/spicylemonhaha/webpack-demo

**What is the expected behavior?**
I think it's right to generate column mapping without babel. One thing to mention here is that although the mappings value in the map file does include column mappings, the column error message is not correctly reported on the browser.

**Other relevant information:**
webpack version:  5.65.0
Node.js version: v14.16.0
Operating System: win10
Additional tools:babel

**Comment by alexander-akait** [comment]
We generated columns, look at generated source map, please clarify what is normal for you?

**Reply by spicylemonhaha** [reply]
I think the normal situation is like this: https://s3.bmp.ovh/imgs/2021/12/5778cf7dd2c26d35.png
However, it is now: https://s3.bmp.ovh/imgs/2021/12/cdb919afb4b5b417.png

**Reply by alexander-akait** [reply]
Sorry, I still do not understand you, you provide different source files

**Reply by spicylemonhaha** [reply]
I have related explanations in the sample code repository, in simple terms:
If only `devtool: source-map` and `mode = development` are configured, the result is the latter.

But the result I expect is the former.

But I accidentally discovered that when I configure a simple babel-loader, it will become the former.

**Reply by alexander-akait** [reply]
Columns generated in both cases, with and without `babel-loader`
